internal/db/storage/blob/s3/reader/reader_test.go

```diff
@@ -128,13 +128,13 @@ func TestNew(t *testing.T) {
 func Test_reader_Open(t *testing.T) {
 	type args struct {
 		ctx context.Context
+		key string
```

Comment:
why did you add the value of key in your test case ?

Reply:
Because of `reader.Reader` interface was modified.
so we have to add the argument of `key`.
Could you please read the bellow link?
https://github.com/vdaas/vald/blob/277f6d945b5ea155a65c18c1aa3003114098fd1f/internal/db/storage/blob/s3/reader/reader.go#L56

Reply:
So?
I mentioned you should set the value of key in your test case.
Does your comment relate my comment? I'm not sure.

Reply:
No problem!
thank you